.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,38 @@
-## Context
+## Context 📙
 
 <!--- Why is this change required? What problem does it solve? -->
 
-## Types of changes
+## Types of changes 🟢 🔶 🟥
```

Comment:
To avoid any ambiguity please clarify what there emojis convey

Reply:
They were to indicate the different types of changes:
- green = safe change (introduces no side-effects)
- yellow = partially safe change (might introduce minor side-effects but not major)
- red = breaking or unsafe change (will introduce major side-effects)

Reply:
how does 🚀 seem instead? I've updated the description to reflect it in the PR doc visualised